sensor/common/centralclient/grpc_connection.go

```diff
@@ -73,6 +74,18 @@ func (f *centralConnectionFactoryImpl) pingCentral() error {
 	return err
 }
 
+func (f *centralConnectionFactoryImpl) getCentralGRPCPreferences() (*v1.Preferences, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+
+	p, err := f.httpClient.GetGRPCPreferences(ctx)
+	if err != nil {
+		return nil, err
+	}
+	return p, nil
```

Comment:
As we are not wrapping the error, then maybe this could be simply 
```suggestion
	return f.httpClient.GetGRPCPreferences(ctx)
```
?

Reply:
Done